sui/wormhole/sources/test/wormhole_scenario.move

```diff
@@ -0,0 +1,145 @@
+#[test_only]
```

Comment:
Maybe worth adding comment saying "this module contains helpers for setting up Sui test scenarios involving Wormhole", since scenarios seem unique to Sui.

Reply:
Added a comment below. How does this look?

Reply:
looks great